src/lib.rs

```diff
@@ -36,3 +36,18 @@
         unused_qualifications, unused_results, variant_size_differences)]
 #![allow(box_pointers, fat_ptr_transmutes, missing_copy_implementations,
          missing_debug_implementations)]
+
+#![allow(dead_code, unused)] // TODO(Spandan) Remove this once code base in sufficient
+
+extern crate routing;
+extern crate sodiumoxide;
+// extern crate safe_dns;
+// extern crate safe_nfs;
+#[macro_use] extern crate log;
+#[macro_use] extern crate safe_core;
+
+mod util;
```

Comment:
This module should not exist yet. The validation is always the matter of frontend. The libraries should not put restrictions as front end is prone to change but libraries are meant to be more stable. All Core should care about are the type-safety. The value should not be core's concern.